src/app-config/service.ts

```diff
@@ -91,6 +93,7 @@ export default class AppService {
       const { token_type, access_token } = this.auth.auth_results;
       this._api.defaults.headers = {
         Authorization: `${token_type} ${access_token}`,
+        'Cli-Version': this.version,
```

Comment:
Should this be `X-Cli-Version` since this is a custom header?

Reply:
I had been unsure about that at first, but sources say the `X-` prefix is basically deprecated

https://specs.openstack.org/openstack/api-wg/guidelines/headers.html

Reply:
Wow... would you look at that! Good call, and good to know.